spec/services/movies_service_spec.rb

```diff
@@ -21,10 +21,10 @@
   it 'can search for movies by keyword in title', :vcr do
     movies = MoviesService.new.search('Inception')
     expect(movies).to be_an(Array)
-    expect(movies.first).to have_key('title')
+    expect(movies.first).to have_key(:title)
```

Comment:
Can we test that the keyword shows up inside the title and it isn't just returned random movies?

Reply:
Made this change.

Reply:
ROCKSTAR